### DISCUSSION TITLE
Add touchpad gestures

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#218] [Ideas]
I think add at least two finger gestures pinch out and pinch for zoom in and zoom out.
add more gestures as four fingers swipe left to open special workspaces 

At this time only three gestures are work to swipe left and right to access the next and previous workspaces

**Comment by prasanthrangan** [comment]
Hi, are these gestures standard in DE's like plasma/gnome? and does it work properly in Hyprland??

**Reply by mahaveergurjar** [reply]
No plasma/gnome gesture is not works in Hyprland only one gesture works in hypland is swiping three fingers left and right to change workspace and already given in the config file That is why working

**Reply by prasanthrangan** [reply]
oh cool, can you try adding it in `hyprland.conf` and test if its works, i'll commit it.

**Comment by mahaveergurjar** [comment]
I think you understand wrong
I am saying the  three finger swipe gesture is already  in hyprland.config 
I am not add and the three finger swipe gesture work. I am saying how to add more gestures 
![16950656971944031337488144564066](https://github.com/prasanthrangan/hyprdots/assets/57872034/09a48d91-36ae-47ec-b21b-73df1933d535)

**Reply by prasanthrangan** [reply]
Oh oki, I'm not sure about it either..please check the wiki to see if it can be configured!

**Reply by mahaveergurjar** [reply]
Okay
